recipe/meta.yaml

```diff
@@ -117,6 +117,7 @@ outputs:
         - mpiexec.sh
         - tests/helloworld.f
         - tests/helloworld.f90
+        - tests/helloworld-f08.f90
```

Comment:
Why don't you just use a `.f08` extension? IIRC, gfortran in conda-forge is fairly new and it should naturally recognize the extension [[docs]](https://gcc.gnu.org/onlinedocs/gfortran/GNU-Fortran-and-GCC.html).

Reply:
I just used the common Fortran practice.. Almost all Fortran projects I am aware of use `.f` for fixed formatted and `.f90` for free formatted source code, independent of the standard revision number their features require.

Reply:
Please note that in this very particular case we may want to somehow state the Fortran version. You are even doing that in the name of the file. I'm just suggesting to do it in a slightly different way via the `.f08` extension. Of course, you are as free to ignore my preference now as I am to ignore yours next time I have to do maintenance in this recipe :smile:.

Reply:
Sure, no problem, I am not dogmatic about this, as long it is not my project. :wink: I've renamed it accordingly.

Reply:
FWIW, I also use `.f90` in my own projects :rofl:.

> Those are my principles, and if you don't like them... well, I have others.
>
> Groucho Marx